docs/config-file-resolutions-yml.md

```diff
@@ -0,0 +1,94 @@
+# The `resolutions.yml` file
+
+Resolutions allow *resolving* any errors or policy violations by marking them as acceptable and can be ignored.
+
+You can re-use [examples/resolutions.yml](examples/resolutions.yml) as the base file for your scans.
+
+### When to Use Resolutions
+Resolutions should be used when it impossible to solve an issue or is planned to be fixed at a later time.
+You can re-use [./examples/resolutions.yml](examples/resolutions.yml) as the base file for your resolutions.
+
+The sections below explains how to create resolutions in the `resolutions.yml` file
+which if passed as an argument to the `reporter` apply to each scan made. If a resolution is project-specific then add it in the [.ort.yml](config-file-ort-yml.md) file for the project.
+
+Resolutions are only taken into account the `reporter`, while the `analyzer` and `scan` results ignore them.
+
+## Resolution Basics
+
+A resolution is applied to specific errors or violations via the regular expression specified
+in the `message` of a resolution.
+
+To be able to show why a resolution is acceptable, each resolution must include an explanation. 
+The explanation consists of:
+
+* `reason` -- quick indicator why the issue was resolved. Must be selected from a predefined list of options. 
```

Comment:
quick indicator why the issue was resolved. Must be selected from a predefined list of options --> identifier selected from a predefined list of options

Reply:
done